### DISCUSSION TITLE
Desreferência de uma referência possivelmente nula.

### DISCUSSION THREAD

**Discussion by franciscoguerra** [balta-io/forum#60] [Tira Dúvidas] [answered]
Surgiu esse erro, até o momento não consegui solucionar. 

![Captura de tela 2022-10-20 002814](https://user-images.githubusercontent.com/12381369/196849639-8ba5ff96-6da8-4f25-bb42-e86fe05b44a2.png)

**Comment by andrebaltieri** [comment, marked as answer]
Boa noite, @franciscoguerra, você salvou o arquivo antes de tentar executar? Parece que não está salvo.

**Comment by williamdev93** [comment]
Estou com esse problema também (eu salvei o arquivo antes de executar)

![image](https://user-images.githubusercontent.com/77591475/228141330-f375e957-408a-4a07-af45-f904419cd697.png)

**Reply by andrebaltieri** [reply]
@williamdev93 este vídeo vai te ajudar:
https://www.youtube.com/watch?v=PNasRBbo00g&t=934s
